sensors/sensors.go

```diff
@@ -22,16 +25,37 @@ type Lidar struct {
 	lidar camera.Camera
 }
 
-// TimedSensorReadingResponse represents a sensor reading with a time & allows the caller to know if the reading is from a replay sensor.
-type TimedSensorReadingResponse struct {
+// IMU represents an IMU movement sensor.
+type IMU struct {
+	Name string
+	imu  movementsensor.MovementSensor
+}
+
+// TimedLidarSensorReadingResponse represents a lidar sensor reading with a time &
+// allows the caller to know if the reading is from a replay camera sensor.
+type TimedLidarSensorReadingResponse struct {
 	Reading     []byte
 	ReadingTime time.Time
 	Replay      bool
 }
 
-// TimedSensor desribes a sensor that reports the time the reading is from & whether or not it is from a replay sensor.
-type TimedSensor interface {
-	TimedSensorReading(ctx context.Context) (TimedSensorReadingResponse, error)
+// TimedLidarSensor describes a sensor that reports the time the reading is from & whether or not it is from a replay sensor.
+type TimedLidarSensor interface {
+	TimedLidarSensorReading(ctx context.Context) (TimedLidarSensorReadingResponse, error)
+}
+
+// TimedIMUSensorReadingResponse represents an IMU sensor reading with a time & allows the caller to know if the reading is
```

Comment:
see above comment

Reply:
Wasn't me that used it originally - want me to make the change or not?